blog-post.html

```diff
@@ -1,12 +1,76 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
-</html>
+
+<head>
+  <title>Blog Post</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+<style>
+  h1 {
+    text-align: center;
+    font: size 60px;
+    font-family: "Notable", sans-serif;
+  }
+
+  .home-btn {
+    background-color: purple;
+    color: white;
+    cursor: pointer;
+  }
+
+  .home-btn:active {
+    color: blue
+  }
+
+  .home-btn:hover {
+    background-color: rgb(63, 61, 61);
+  }
+</style>

```

Comment:
We can put all of this into the CSS file to keep things a little more organized, then Link the CSS file to this file

Reply:
yea i am going to move it over i was just giving it some basic styling and i was just going to copy and paste it over and make  modifications. i just wasted so much time with flex box that i pieced all the other stuff together really quickly